tutorials/last_value_cache/c++11/home_automation_audited.cxx

```diff
@@ -0,0 +1,44 @@
+//
+// (c) 2024 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+//
+//  RTI grants Licensee a license to use, modify, compile, and create derivative
+//  works of the Software.  Licensee has the right to distribute object form
+//  only for use with RTI products.  The Software is provided "as is", with no
+//  warranty of any type, including any warranty for fitness for any purpose.
+//  RTI is under no obligation to maintain or support the Software.  RTI shall
+//  not be liable for any incidental or consequential damages arising out of the
+//  use or inability to use the software.
+//
+
+#include <iostream>
+#include <thread>
+#include <rti/rti.hpp>
+#include "home_automation.hpp"
+
+void publish_sensor(
+    const std::string& sensor_name,
+    const std::string& room_name)
+{
+    dds::domain::DomainParticipant participant(0);
+    dds::topic::Topic<DeviceStatus> topic(participant, "WindowStatus");
+    dds::pub::DataWriter<DeviceStatus> writer(
+            topic,
+            dds::core::QosProvider::Default().datawriter_qos("Last10Cache"));
```

Comment:
Should we create a unique publisher with a history of 10? And just modify the subscriber/audit file?

Reply:
good question - I don't like having two publishers, but I also don't want to distract from the first part of the module by always setting the history to 10.

🤔

Reply:
I think we should update the instructions in the guide to ask the user to manually modify the publisher application to use the new profile when a publisher needs to be audited, and delete the extra publisher file from this PR.

Reply:
Sounds good to me :+1: